plugins/indentlist/plugin.js

```diff
@@ -174,7 +174,19 @@
 				listArray[ i ].indent += indentOffset;
 				// Make sure the newly created sublist get a brand-new element of the same type. (http://dev.ckeditor.com/ticket/5372)
 				if ( indentOffset > 0 ) {
-					var listRoot = listArray[ i ].parent;
+					var listRoot;
```

Comment:
While further in the code you are checking:

```
if ( listRoot === undefined ) {
    listRoot = listArray[ i ].parent;
}
```
why not initialize `var listRoot = listArray[ i ].parent;` so it already has default value which alternatively will be overwritten inside the `for` loop.